src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/OptionMetadata.php

```diff
@@ -32,7 +32,7 @@ class OptionMetadata
     protected $type;
 
     /**
-     * @var string|int|OptionMetadata[]
+     * @var string|float|int|OptionMetadata[]
```

Comment:
was adjusted by @draconivis in: https://github.com/alexander-schranz/sulu/pull/8#event-17631157226

we need to recheck if "new metadata" ever writes as a value `float` if not we need adjust the test that such data comes as `'1.5'` and maybe need to parse it to a float value.